empiric-package/empiric/core/utils.py

```diff
@@ -1,5 +1,22 @@
+import logging
 import warnings
 
+logger = logging.getLogger(__name__)
+
+
+def get_stream_logger():
+    import sys
+
+    logger.setLevel(logging.DEBUG)
+    streamHandler = logging.StreamHandler(sys.stdout)
+    formatter = logging.Formatter(
+        "%(asctime)s - %(name)s - %(levelname)s - %(message)s"
+    )
+    streamHandler.setFormatter(formatter)
+    logger.addHandler(streamHandler)
+
+    return logger
+
```

Comment:
Could see this in a separate logging-specific file outside utils as well

Reply:
sure, i can move that to `logger.py`